BubbleScan-AI/result_data.json

```diff
@@ -0,0 +1,1720 @@
+{
```

Comment:
Did you intentionally commit this file or was this by accident?

Reply:
Yes, this can be helpful for someone who wants to look at the JSON data format for converting the output to csv.